src/Aggregation/Metric/PercentileRanksAggregation.php

```diff
@@ -114,28 +81,20 @@ public function getArray()
                 'values' => $this->getValues(),
                 'compression' => $this->getCompression(),
             ],
-            function ($val) {
-                return ($val || is_numeric($val));
-            }
+            fn(mixed $val): bool => $val || is_numeric($val)
         );
 
         $this->isRequiredParametersSet($out);
 
         return $out;
     }
 
-    /**
-     * @param array $a
-     *
-     * @return bool
-     * @throws \LogicException
-     */
-    private function isRequiredParametersSet($a)
+    private function isRequiredParametersSet(array $a): void
```

Comment:
The previous implementation returned `bool`, why did you changed to `void`?

Reply:
fixed